blocks/hero-carousel/hero-carousel.js

```diff
@@ -0,0 +1,125 @@
+import {
+  addEnclosingDiv,
+  createDottedNav,
+  createSlider,
+  createSliderWrapper,
+  initSlider,
+  setSliderIds,
+} from '../../scripts/lib-carousel.js';
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+
+// Number of required columns in table
+const NUM_COLUMNS = 3;
+
```

Comment:
as the same error message is used multiple times, suggest introducing a constant:
```
const INVALID_CONFIGURATION_MESSAGE = `Invalid configuration. Table with ${NUM_COLUMNS} columns and at least 1 row required`;
```